src/App.jsx

```diff
@@ -1,35 +1,33 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
 
-function App() {
-  const [count, setCount] = useState(0)
+const todoList = [
+  {
+    id:1,
+    title:"Walk Dogs"
+  },
+  {
+    id:2,
+    title:"Clean Room"
+  },
+  {
+    id:3,
+    title:"Take Trash Out"
+  },
+];
+
+/* rendering the list */
+const renderList = todoList.map(function(listItem){
+  return <li>{listItem.id}: {listItem.title}</li>
```

Comment:
Remember to add a unique key to each element when rendering lists in React. This allows React to efficiently handle updates, insertions, and removals within the list.